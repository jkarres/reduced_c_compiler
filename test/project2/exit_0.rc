
function : int doexit(int i) {
  exit(i);
  return 12345;
}

function : void main() {
  cout << "should exit with 0" << endl;
  cout << "this should be the last line you see" << endl;
  cout << doexit(0) << endl;
  cout << "your test failed." << endl;
}
