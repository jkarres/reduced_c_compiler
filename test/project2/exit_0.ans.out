should exit with 0
this should be the last line you see
